### DISCUSSION TITLE
cannot override Option component

### DISCUSSION THREAD

**Discussion by zgid123** [JedWatson/react-select#5570] [Q&A]
Hi,

I am trying to customize the `react select` components, but it seems the `Option` is not passed to the `getComponents`.

```ts
    <ReactSelect
      components={{
        Option,
        DropdownIndicator,
        IndicatorSeparator: null,
      }}
    />
```

**Comment by souvik666** [comment]
hey @zgid123  please try like this 

```
const Option = (props) => {
  const {
    ...
    data,
  } = props
  return (
    <div ...>
      {`${data.firstName} - ${data.lastName}`}
    </div>
  )
}

<Select
    ...
    components={{ Option }}
    ...
  />
```

**Reply by zgid123** [reply]
Thanks for your effort. This's been a long time, I remember at that time, I imported a wrong component or something like that. I already fixed the problem but don't remember how.

Thanks again.
